grin/src/Grin/Syntax/Extended.hs

```diff
@@ -0,0 +1,149 @@
+{-# LANGUAGE DeriveDataTypeable, DeriveGeneric, DeriveAnyClass, DeriveFunctor, TypeFamilies #-}
+{-# LANGUAGE DeriveFoldable, DeriveTraversable, PatternSynonyms #-}
+{-# LANGUAGE TemplateHaskell, StandaloneDeriving, OverloadedStrings #-}
+{-# LANGUAGE LambdaCase #-}
+module Grin.Syntax.Extended
+  ( module Grin.Syntax.Extended
+  , module Grin.SyntaxDefs
+
+  -- reexports from Grin.Syntax
+  , Ty(..)
+  , ExternalKind(..)
+  , External(..)
+  , Lit(..)
+  , CPat(..)
+  , _CPatNodeTag
+  , _TyCon
+  , _TyVar
+  , _TySimple
+  , _CPatLit
+  , _CPatDefault
+  , isExternalName
+  ) where
+
+import Data.Data
+import Control.DeepSeq
+import Data.Binary
+import Data.Functor.Foldable.TH
+import Data.Int
+import Data.Text (Text, isPrefixOf)
+import Data.Vector
+import Data.Word
+import GHC.Generics (Generic)
+import Lens.Micro.Platform
+import qualified Data.ByteString.Short as B
+
+import Grin.SyntaxDefs
+import Grin.TypeEnvDefs
+
+import Grin.Syntax
```

Comment:
I would prefer the qualified import over the long import list. What is the rational not importing everything from the module or adding it as qualified one?

Reply:
I added the names incrementally, will resolve that.

Reply:
Please link the commit hash here when this is got resolved.

Reply:
Here's the commit: 977c31b666c7cc127edb7a85cff7d4ceb56d53e5